wporg-two-factor.php

```diff
@@ -35,9 +35,14 @@ function two_factor_providers( array $providers ) : array {
 	$desired_providers = array(
 		'Two_Factor_WebAuthn'     => '',
 		'Two_Factor_Totp'         => '',
-		'Two_Factor_Backup_Codes' => '',
 	);
 
+	// Check current user has a primary method setup, before allowing backup codes
+	$user_providers = Two_Factor_Core::get_enabled_providers_for_user();
```

Comment:
```suggestion
	$user_providers = Two_Factor_Core::get_available_providers_for_user();
```

i didn't realize that this until recently, but the enabled/available functions are confusingly named. A provider can be "enabled", but not it's "available" until it's been _configured_. I think in this context we want the latter.